src/pages/Metrics/components/MetricsInfo/MetricsInfo.tsx

```diff
@@ -0,0 +1,224 @@
+import CircleProgressBar from '../../../../components/CircleProgressBar/CircleProgressBar';
+import {
+  associationsArray, colorsArray, emotionsArray, objectsArray,
+} from '../../MetricsTestValues.data';
+
+function MetricsInfo({
+  isLoading,
+}: {
+  isLoading: boolean;
+}) {
+  const UNIQ = 52;
+  const Colors = 20;
+  const Other = 15;
```

Comment:
not clear why constants are named using different naming conventions - please either use camelCase or (preferably) UPPERCASE approach

also you need to move them to the module level, otherwise they'll be created on each render cycle